src/Esfa.Vacancy.Api.Types/ApprenticeshipVacancy.cs

```diff
@@ -212,14 +212,18 @@ public class ApprenticeshipVacancy
         /// </summary>
         public string ApprenticeshipLevel { get; set; }
 
-        /// <summary>
-        /// Instructions on how to apply for the vacancy
-        /// </summary>
-        public string ApplicationInstructions { get; set; }
-
-        /// <summary>
-        /// The web URL for applying for the vacancy
-        /// </summary>
-        public string ApplicationUrl { get; set; }
-    }
+		  /// <summary>
+		  /// Instructions on how to apply for the vacancy
+		  /// </summary>
+		  public string ApplicationInstructions { get; set; }
+
+		  /// <summary>
+		  /// The web URL for applying for the vacancy
```

Comment:
`/// The web URL for applying for the vacancy outside of Find an Apprenticeship` maybe

Reply:
Those were the original comments